### ISSUE TITLE
Files won't process in development environment without ClamAV installed

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/laevigata#1320]
**ISSUE**
I'm not able to get files to attach in my development environment, and I'm seeing the following error in Sidekiq for every file I've tried to attach:

```
Exceptions::ClamscanMissing: clamscan application not found. Check your installation and path.
```

### ACCEPTANCE
- [ ] Update the README with appropriate instructions to disable scanning or to install ClamAV

<img width="1436" alt="1320 - clamav in development" src="https://user-images.githubusercontent.com/3064318/42738517-7bfeae6a-884a-11e8-9b4e-8b6c57f81146.png">

**Comment by mark-dce** [comment]
I just installed ClamAV (using homebrew) and this error has cleared.  We should talk about whether this is a desired behavior - if so, we should mention it in the README as a development dependency.

**Comment by mark-dce** [comment]
Addressed in PR #1924
